main.py

```diff
@@ -1,10 +1,11 @@
-# initialize the amount variable
```

Comment:
Please find the requried dependencies and fix them.

['**', '', 'requirements.txt', '', 'pytest==3.2.1']